workspaces/bulk-import/plugins/bulk-import/src/types/types.ts

```diff
@@ -97,7 +97,17 @@ export enum RepositoryStatus {
   'REPO_EMPTY' = 'REPO_EMPTY',
   'CODEOWNERS_FILE_NOT_FOUND_IN_REPO' = 'CODEOWNERS_FILE_NOT_FOUND_IN_REPO',
 }
-
+export enum AddedRepositoryColumnNameEnum {
+  repoName = 'repository.name',
+  organizationUrl = 'repository.organization',
+  repoUrl = 'repository.url',
+  catalogInfoYamlLastUpdated = 'lastUpdate',
+  catalogInfoYamlStatus = 'status',
+}
+export enum SortingOrderEnum {
+  ASC = 'asc',
+  DESC = 'desc',
+}
```

Comment:
Please respect the convention in this file

* and add empty lines between each type definition
* and it looks like enum keys here are all UPPER_CASE